bitnami/zookeeper/README.md

```diff
@@ -206,6 +194,25 @@ The following tables lists the configurable parameters of the ZooKeeper chart an
 | `metrics.prometheusRule.selector`      | Prometheus instance selector labels                                                                                                       | `nil`                                                        |
 | `metrics.prometheusRule.rules`         | Prometheus Rule definitions (see values.yaml for examples)                                                                                | `[]`                                                         |
 
+### TLS/SSL parameters
+
+| Parameter                        | Description                                                                                    | Default                                                               |
+|----------------------------------|------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|
+| `tls.client.enabled`             | Enable TLS for client connections                                                              | `false`                                                               |
+| `tls.client.autoGenerated`       | Generate automatically self-signed TLS certificates for Zookeeper client communications        | `false`                                                               |
```

Comment:
The auto-generated TLS certs will be using PEM format (instead of JKS). We should mention it.